### DISCUSSION TITLE
Multi profile (VPN Config)

### DISCUSSION THREAD

**Discussion by BliNeR-KeY** [d3vilh/openvpn-ui#28] [Ideas]
Is only idea nothing issue..

Its possible add more profile authentication username/password
At last openvpn-ui have own dB. 
Without using extra openvpn plugin 

Like surfshark..
Same one profile same cert.. Same key..
Only different username/password authorization?

**Comment by d3vilh** [comment]
It was implemented in [version 0.9.2](https://github.com/d3vilh/openvpn-ui/releases/tag/0.9.2) as permanent change for all the future versions as well.

[Documentation was updated ](https://github.com/d3vilh/openvpn-ui?tab=readme-ov-file#user-management)to include all the user management details.
